app/src/Homepage/Homepage.tsx

```diff
@@ -1,19 +1,20 @@
-import {Header, Sidebar, LevelTab, InnerPage} from './Homepage-components';
+import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
+import {InnerPage, HomeLayout} from './Homepage-components';
 import './Homepage.css';
 
 function App() {
   return (
-    <div className='App'>
-      <Header/>
-      <Sidebar>
-        <LevelTab classlevel='Home'/>
-        <LevelTab classlevel='CSE 100s'/>
-        <LevelTab classlevel='CSE 300s'/>
-        <LevelTab classlevel='CSE 400s'/>
-        <LevelTab classlevel='CSE 500s'/>
-      </Sidebar>
-      <InnerPage/>
-    </div>
+    <Router>
+      <HomeLayout>
+        <Routes>
+          <Route path='rate-my-cse/' element={<InnerPage/>} />
```

Comment:
Change from using hard coded path to be more dynamic. (Check above comments)

Reply:
Changed